src/auth/auth.controller.ts

```diff
@@ -19,7 +20,7 @@ export default class AuthController {
     res.status(StatusCodes.OK).json({ message: "It works!" });
   }
 
-  async logout(req: Request, res: Response) {
+  async logout(req: AuthenticatedRequest, res: Response) {
```

Comment:
With having this type in controller and auth middleware for the current route, one can rely on `req.user` being set.